apps/prairielearn/src/middlewares/logPageView.ts

```diff
@@ -44,7 +39,7 @@ export async function logPageView(pageType, req, res) {
   });
 }
 
-export default function (pageType) {
+export default function (pageType: string) {
```

Comment:
Nit: consider defining a more precise type for this. There's a finite number of these that we want to use; defining a specific type would help us catch typos in the future.